extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -471,13 +476,35 @@ export class AmpStoryPage extends AMP.BaseElement {
   }
 
   /**
-   * Finds embedded components in page and prepares them for their expanded view
-   * animation.
+   * Finds embedded components in page and prepares them.
    * @param {boolean=} forceResize
    * @private
    */
   findAndPrepareEmbeddedComponents_(forceResize = false) {
-    scopedQuerySelectorAll(this.element, EMBEDDED_COMPONENTS_SELECTORS)
+    this.addClickShieldToEmbeds_();
+    this.resizeInteractiveEmbeds_(forceResize);
```

Comment:
Naming nit: `embed` could be anything really (any content within the page), could we use EmbeddedComponents again?